### ISSUE TITLE
Multiple templates 

### ISSUE THREAD

**Issue by alexseys** [bzon/prometheus-msteams#79]
Is there a way to use different templates for different channels?

**Comment by Knappek** [comment]
Unfortunately this is not yet supported. It's a valid feature request though, feel free to pick it up and open a PR :).

**Comment by alexseys** [comment]
Thank you!
Sorry, not a developer by any means.

**Comment by Knappek** [comment]
No worries, it's a valid feature request, let's reopen it.

**Comment by bzon** [comment]
I think this feature can be much easier to add with the latest code structure.

```yaml
connectors:
- request_path: /uri
   webhook : <teams webhook>
   template_file: /path/to/file
```

See also the use of  **converter** per each route.
https://github.com/bzon/prometheus-msteams/blob/24b1f51738c00f1aa7e702f437c2823ea854bc7c/cmd/server/main.go#L158-L167

**Comment by bzon** [comment]
Fixed in #91 and in v1.3.0
